cpmpy/expressions/globalfunctions.py

```diff
@@ -42,8 +42,8 @@ class my_global(GlobalFunction):
             def __init__(self, args):
                 super().__init__("my_global", args)
 
-            def decompose_comparison(self):
-                return [self.args[0] + self.args[1]] # your decomposition
+            def decompose(self):
+                return [self.args[0] + self.args[1]], [] # your decomposition
```

Comment:
I guess this global is "MySum" or so, and then this should now be `return (self.args[0]+self.args[1]), []`